q2_vsearch/_cluster_features.py

```diff
@@ -177,4 +279,48 @@ def cluster_features_closed_reference(sequences: DNAFASTAFormat,
                            axis='observation',
                            include_collapsed_metadata=False)
 
-    return table, unmatched_seqs
+    return table, matched_seqs, unmatched_seqs
```

Comment:
It looks like there is a bug in the `matched_seqs` output, where some sequences can be reported multiple times. @thermokarst and I are communicating offline about this, but just wanted to add a note for the record here.

Reply:
Confirmed that this bug is no longer present.